docs/reference/release-notes/7.5.asciidoc

```diff
@@ -1,7 +1,77 @@
+[[release-notes-7.5.1]]
+== {es} version 7.5.1
+
+Also see <<breaking-changes-7.5,Breaking changes in 7.5>>.
+
+[[enhancement-7.5.1]]
+[float]
+=== Enhancements
+
+Features/Watcher::
+* Log attachment generation failures {pull}50080[#50080]
+
+Network::
+* Netty4: switch to composite cumulator {pull}49478[#49478]
+
+
+
+[[bug-7.5.1]]
+[float]
+=== Bug fixes
+
+Authentication::
+* Fix iterate-from-1 bug in smart realm order {pull}49473[#49473]
+
+CRUD::
+* Do not mutate request on scripted upsert {pull}49578[#49578] (issue: {issue}48670[#48670])
+
+Cluster Coordination::
+* Make elasticsearch-node tools custom metadata-aware {pull}48390[#48390]
+
+Features/ILM+SLM::
+* Handle failure to retrieve ILM policy step better {pull}49193[#49193] (issue: {issue}49128[#49128])
```

Comment:
```suggestion
* Handle failure to retrieve ILM policy step better {pull}49193[#49193] (issue: {pull}49128[#49128])
```

This isn't actually an linked issue but another pull request. I'd consider removing it. If you decide to leave it in, I'd use the `{pull}` attribute to avoid an unneeded redirect.